fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java

```diff
@@ -186,4 +194,67 @@ public boolean parse(String value) {
             throw new RuntimeException("string check failed ", e);
         }
     }
+}
+
+class DecimalParser extends ColumnParser {
+
+    public static int PRECISION = 27;
+    public static int SCALE = 9;
+
+    private BigDecimal maxValue;
+    private BigDecimal minValue;
+
+    public DecimalParser(EtlJobConfig.EtlColumn etlColumn) {
+        StringBuilder precisionStr = new StringBuilder();
+        for (int i = 0; i < etlColumn.precision - etlColumn.scale; i++) {
```

Comment:
Here you can write a map to enumerate all the combinations to improve efficiency.